server/predictions.py

```diff
@@ -0,0 +1,52 @@
+import sys
+sys.path.append("../")
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import firestore
+import server.auth as auth
+import datetime
+
+# Fill in as outcomes come in? Keys should be form fields, values should all be either 0 or 100
+realized_outcomes = {}

```

Comment:
Can we not just have the realized outcome be a property that we have in the yaml?